gruntfile.js

```diff
@@ -34,29 +32,74 @@ module.exports = function(grunt) {
     });
   });
 
+  var modifyDefaultConfig = function(confName, jsdocOptions, mutatorFn){
+    var confPath = path.join("jsdocs", confName);
+    if(fs.existsSync(confPath)){
+      fs.unlinkSync(confPath);
+    }
+    var defaultConf = JSON.parse(fs.readFileSync(jsdocOptions.configure));
+    fs.writeFileSync(confPath, JSON.stringify(mutatorFn(defaultConf)));
+    return confPath;
+  }
+
   var jsdocConfig = {};
   _.each(sources, function(source) {
+    var examplesPath = path.join(path.dirname(source[1]), "examples");
+    var docOptions = _.clone(jsdocOptions);
+
+    fs.exists(examplesPath, function(exists){
```

Comment:
i think we should use "existsSync" here; otherwise, the code might not run correctly